content/applications/hr/payroll/contracts.rst

```diff
@@ -2,224 +2,361 @@
+For an employee to be paid, an active contract is required. When a new contract is needed, click the
+:guilabel:`New` button on the contract dashboard. A contract form appears where the information can
+be entered.
 
 Required fields
 ---------------
 
-.. image:: contracts/required-fields.png
-   :align: center
-   :alt: New contract form to be filled in when creating a new contract.
-
-- :guilabel:`Contact Reference`: Type in the name or title for the contract, such as `John Smith
+- :guilabel:`Contact Reference`: type in the name or title for the contract, such as `John Smith
   Contract`.
-- :guilabel:`Company`: Select the company the contract applies to by clicking on the drop-down menu.
-  A new company can be created by typing the name in the field, then clicking either
-  :guilabel:`Create` to create the new company, or :guilabel:`Create and Edit` to create the new
-  company and edit the company details.
-- :guilabel:`Salary Structure Type`: Select one of the salary structure types from the drop-down
-  menu. The default salary structure types are :guilabel:`Employee` or :guilabel:`Worker`. A new
-  salary structure type can be created by typing the name in the field.
-- :guilabel:`Start Date`: The date the contract starts. Choose a date by clicking on the drop-down
-  menu, navigating to the correct month and year by using the :guilabel:`< > (arrow)` icons, then
-  clicking on the :guilabel:`date`.
-- :guilabel:`Working Schedule`: Select one of the working schedules from the drop-down menu.
+- :guilabel:`Contract Start Date`: the date the contract starts. To choose a date, click the
+  drop-down menu, navigate to the correct month and year with the :guilabel:`< > (arrow)` icons,
+  then click on the :guilabel:`date`.
+- :guilabel:`Working Schedule`: select one of the working schedules from the drop-down menu.
 
 .. tip::
-   The :guilabel:`Working Schedule` drop-down menu displays all the working times for the selected
-   :guilabel:`Company`. To modify or add to this list, go to :menuselection:`Payroll -->
-   Configuration --> Working Times` and either :guilabel:`Create` a new working time or click on an
-   existing working time and edit it by clicking :guilabel:`Edit`.
+   The :guilabel:`Working Schedule` drop-down menu displays all the working schedules for the
+   selected :guilabel:`Company`. To modify or add to this list, go to :menuselection:`Payroll app
+   --> Configuration --> Working Schedules`. Click :guilabel:`New` and create a new working
+   schedule, or click on an existing working schedule and make edits.
+
+- :guilabel:`Work Entry Source`: select how the :doc:`work entries <../../hr/payroll/work_entries>`
+  are generated. Click the radio button next to the desired selection. The options are:
+
+  - :guilabel:`Working Schedule`: work entries are generated based on the selected
+    :guilabel:`Working Schedule`.
+  - :guilabel:`Attendances`: work entries are generated based on the employee's check in records in
```

Comment:
```suggestion
  - :guilabel:`Attendances`: work entries are generated based on the employee's check-in records in
```
(I like this better than "attendances attendances attendances" :laughing: )

Reply:
And here I could have sworn it was without a hyphen =D One day I'll get it right!